vphysics_jolt/vjolt_objectpairhash.cpp

```diff
@@ -71,7 +76,11 @@ void JoltPhysicsObjectPairHash::RemoveAllPairsForObject( void *pObject0 )
 
 bool JoltPhysicsObjectPairHash::IsObjectInHash( void *pObject0 )
 {
+#if __cplusplus >= 202002L
     return m_Objects.contains( pObject0 );
+#else
+	return m_Objects.find( pObject0 ) != m_Objects.cend();
```

Comment:
Shouldn't this be just .end()?

Reply:
i mean it doenst matter, the funcs should have been marked as const but they arent so i used cend